### DISCUSSION TITLE
Allow elements to be dragged into other matrix fields

### DISCUSSION THREAD

**Discussion by Nellyaa** [craftcms/cms#15880] [Ideas] [answered]
In addition to disabling, duplicating & copy/paste, it'd be great if we could drag our elements from a matrix field into another matrix field that allows this entry type.
In my case this'd be a big time saver!

![bsp](https://github.com/user-attachments/assets/83ee9f47-1729-47ab-9be4-f4d8082e5aa9)

**Comment by alexanderbuergin** [comment]
+1 – It should not only be possible to move Craft CMS blocks within a single Matrix field, but also across different Matrix fields, provided that the respective entry type is allowed in the Matrix field at the desired position.

**Comment by ryanleichty** [comment]
Ohhh, I was just thinking about this too. This would be really helpful within a page builder context!

**Reply by alexanderbuergin** [reply]
We need this.

**Comment by alexanderbuergin** [comment]
+1

**Comment by brandonkelly** [comment, marked as answer]
Craft 5.7 is adding the ability to copy/paste entries and addresses across sections/fields 🎉 (#16888)

**Reply by brandonkelly** [reply]
Craft 5.7.0 is out with copy-paste support for entries and addresses 🎉
